### PR TITLE
Fix no rounded corners of DialogActionsBox in Withdraw and Trading view

### PR DESCRIPTION
**Pull request by Marcel-Ebert**
- [x] To fix padding issues on iOS wrap the `<DialogActionsBox` with a `<Portal>` component in the withdraw view and with the `<ActionsContainer>` in the trading view
- [x] Slightly reduce the width of the `<DialogActionsBox>` on mobile
- [x] Align the "Add Asset" buttons in trading and withdraw view to the bottom right hand side

### CODE REVIEW

File: src/components/Dialog/Generic.tsx [hunk 1]
```diff
@@ -38,9 +38,9 @@ const useActionButtonStyles = makeStyles(theme => ({
   mobileDialogActionsBox: {
     display: "flex",
     position: "fixed",
-    left: 0,
+    left: 8,
```
Comment (andywer) [inline src/components/Dialog/Generic.tsx]: @Marcel-Ebert Can you remember why we made it 8px left and right, instead of 0 as before?

Comment (Marcel-Ebert) [reply src/components/Dialog/Generic.tsx]: To make it smaller

Comment (andywer) [reply src/components/Dialog/Generic.tsx]: Yeah, but why? 😅
Can't remember anymore...

Comment (Marcel-Ebert) [reply src/components/Dialog/Generic.tsx]: Because you were unhappy with the space between the button(s) and the edge of the screen 🤐

**Pull request merged**